src/helpers.ts

```diff
@@ -0,0 +1,28 @@
+import { Person } from './types';
+
+const getFather = (
+  fatherName: string | null,
+  array: Person[],
+): Person | undefined => {
+  return array.find(father => father.name === fatherName);
+};
+
+const getMother = (
+  motherName: string | null,
+  array: Person[],
+): Person | undefined => {
+  return array.find(mother => mother.name === motherName);
+};
```

Comment:
Can it be one function?
```suggestion
const getParent = (
  parentName: string | null,
  array: Person[],
): Person | undefined => {
  return array.find(person => person.name === parentName);
};
```